src/winetricks

```diff
@@ -10908,7 +10911,7 @@ load_allfonts()
     do
         cmd=$(basename "$file" .vars)
         case $cmd in
-            allfonts|cjkfonts) ;;
+            allfonts|cjkfonts|fake*|pptfonts) ;;
```

Comment:
As this list is growing, please add a comment explaining why they're being skipped.

Other than that, LGTM